migrate/indexerdb0to1.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const assert = require('assert');
+const hash256 = require('bcrypto/lib/hash256');
+const Filter = require('../lib/primitives/filter');
+const layout = require('../lib/indexer/layout');
+const FileBlockStore = require('../lib/blockstore/file');
+const bdb = require('bdb');
+const path = require('path');
+const fs = require('bfile');
+const {filters} = require('../lib/blockstore/common');
+
+assert(process.argv.length > 2, 'Please pass in a database path.');
+
+// change:
+// Move filter data present in ~/.../filter/ to ~/.../filter/BASIC
+
+const location = path.resolve(process.argv[2], '../blocks');
```

Comment:
Have you tested this on Windows? Might need to use `path.join()` so its compatible with all platforms.